packages/playwright-crawler/src/internals/adaptive-playwright-crawler.ts

```diff
@@ -351,7 +359,7 @@ export class AdaptivePlaywrightCrawler extends PlaywrightCrawler {
                         return Reflect.get(target, propertyName, receiver);
                     },
                 }),
-                crawlingContext,
+                crawlingContext as any,
```

Comment:
Why do we need to do this? Care to add a comment?

Reply:
because the restricted context is not compatible with browser context

Reply:
hmm or maybe its just an artifact of previous version, it works fine without it :]